libensemble/gen_funcs/persistent_sampling.py

```diff
@@ -25,10 +35,8 @@ def persistent_uniform(_, persis_info, gen_specs, libE_info):
         `test_persistent_sampling.py <https://github.com/Libensemble/libensemble/blob/develop/libensemble/tests/regression_tests/test_persistent_sampling.py>`_ # noqa
         `test_persistent_sampling_async.py <https://github.com/Libensemble/libensemble/blob/develop/libensemble/tests/regression_tests/test_persistent_sampling_async.py>`_ # noqa
     """
-    ub = gen_specs["user"]["ub"]
-    lb = gen_specs["user"]["lb"]
-    n = len(lb)
-    b = gen_specs["user"]["initial_batch_size"]
+
+    b, n, lb, ub = _get_user_params(gen_specs["user"])
```

Comment:
I like this, maybe we ought to encourage this sort of extraction in the future (or extract user params into *args?)

Reply:
I agree, but couldn't that introduce namespace conflicts if everything is automatically extracted?